src/libcode/vx_statistics/pair_data_point.cc

```diff
@@ -1470,6 +1478,23 @@ void VxPairDataPoint::set_obs_perc_value(int percentile) {
 
 ////////////////////////////////////////////////////////////////////////
 
+void VxPairDataPoint::load_seeps_climo() {
+   bool loaded = false;
+   for(int i=0; i < n_msg_typ; i++){
+      for(int j=0; j < n_mask; j++){
+         for(int k=0; k < n_interp; k++){
+            pd[i][j][k].load_seeps_climo();
+            loaded = true;
```

Comment:
@hsoh-u, I think PairDataPoint::load_seeps_climo() is just setting a pointer to a single instance of the SEEPS climo info. But that pointer should be set for each PairDataPoint object. So I don't think you should be breaking out of the loop based on the "loaded". You could just remove all the "loaded" stuff, as I've recommended here.
```suggestion
```